docs/tech-architecture.svg

```diff
@@ -1 +1 @@
linecap="butt" d="m289.3719 159.98415l1.6517334 4.538086l1.6517334 -4.538086z" fill-rule="evenodd"/><path fill="#000000" fill-opacity="0.0" d="m238.11023 81.538055l105.82678 0l0 18.897636l-105.82678 0z" fill-rule="evenodd"/><path fill="#000000" d="m283.75125 84.323685l1.578125 0l0 5.703125q0 0.9375 -0.4375 1.65625q-0.4375 0.703125 -1.234375 1.109375q-0.796875 0.390625 -1.859375 0.390625q-1.078125 0 -1.875 -0.390625q-0.796875 -0.40625 -1.234375 -1.109375q-0.4375 -0.71875 -0.4375 -1.65625l0 -5.703125l1.578125 0l0 5.578125q0 0.546875 0.234375 0.96875q0.25 0.421875 0.6875 0.671875q0.4375 0.234375 1.046875 0.234375q0.59375 0 1.03125 -0.234375q0.453125 -0.25 0.6875 -0.671875q0.234375 -0.421875 0.234375 -0.96875l0 -5.578125zm8.413116 3.921875l-1.40625 0.15625q-0.0625 -0.21875 -0.21875 -0.40625q-0.140625 -0.1875 -0.390625 -0.296875q-0.234375 -0.125 -0.59375 -0.125q-0.484375 0 -0.8125 0.21875q-0.328125 0.203125 -0.328125 0.53125q0 0.296875 0.21875 0.46875q0.21875 0.171875 0.71875 0.296875l1.109375 0.234375q0.921875 0.203125 1.375 0.640625q0.46875 0.4375 0.46875 1.140625q0 0.609375 -0.359375 1.078125q-0.359375 0.46875 -1.0 0.734375q-0.625 0.265625 -1.4375 0.265625q-1.203125 0 -1.9375 -0.5q-0.734375 -0.5 -0.875 -1.40625l1.5 -0.15625q0.109375 0.453125 0.4375 0.6875q0.34375 0.21875 0.875 0.21875q0.546875 0 0.875 -0.21875q0.34375 -0.234375 0.34375 -0.5625q0 -0.28125 -0.21875 -0.46875q-0.21875 -0.1875 -0.671875 -0.28125l-1.109375 -0.234375q-0.953125 -0.203125 -1.40625 -0.65625q-0.4375 -0.46875 -0.4375 -1.1875q0 -0.609375 0.328125 -1.046875q0.328125 -0.453125 0.921875 -0.6875q0.609375 -0.25 1.390625 -0.25q1.15625 0 1.8125 0.484375q0.671875 0.484375 0.828125 1.328125zm4.306671 4.9375q-0.984375 0 -1.703125 -0.40625q-0.703125 -0.40625 -1.09375 -1.15625q-0.375 -0.765625 -0.375 -1.796875q0 -1.015625 0.375 -1.78125q0.390625 -0.765625 1.078125 -1.1875q0.703125 -0.421875 1.640625 -0.421875q0.609375 0 1.140625 0.1875q0.546875 0.1875 0.953125 0.59375q0.421875 0.40625 0.65625 1.03125q0.234375 0.625 0.234375 1.484375l0 0.484375l-5.359375 0l0 -1.046875l3.890625 0q0 -0.4375 -0.1875 -0.78125q-0.1875 -0.359375 -0.53125 -0.546875q-0.328125 -0.203125 -0.78125 -0.203125q-0.46875 0 -0.828125 0.234375q-0.359375 0.21875 -0.5625 0.59375q-0.1875 0.359375 -0.1875 0.796875l0 0.90625q0 0.578125 0.203125 0.984375q0.203125 0.40625 0.578125 0.625q0.375 0.203125 0.890625 0.203125q0.328125 0 0.59375 -0.09375q0.28125 -0.09375 0.484375 -0.28125q0.203125 -0.1875 0.296875 -0.46875l1.4375 0.171875q-0.125 0.5625 -0.515625 0.984375q-0.375 0.421875 -0.96875 0.65625q-0.59375 0.234375 -1.359375 0.234375zm4.221405 -0.125l0 -6.546875l1.484375 0l0 1.09375l0.078125 0q0.171875 -0.5625 0.609375 -0.875q0.4375 -0.3125 1.0 -0.3125q0.125 0 0.28125 0.015625q0.171875 0 0.28125 0.03125l0 1.421875q-0.109375 -0.046875 -0.328125 -0.0625q-0.203125 -0.03125 -0.390625 -0.03125q-0.421875 0 -0.765625 0.1875q-0.328125 0.171875 -0.53125 0.5q-0.1875 0.3125 -0.1875 0.734375l0 3.84375l-1.53125 0z" fill-rule="nonzero"/></g></svg>
```

Comment:
what changed here as I can still see azure in the final image?

Reply:
that's odd, I didn't chance much but it should look like this...
<img width="584" alt="image" src="https://user-images.githubusercontent.com/450843/117679113-e4ba4700-b1a7-11eb-9588-176eea4667ac.png">

Reply:
bah I think Github might be displaying the old one when I try viewing it in this pr 👍